File: app/src/main/java/com/rsupport/mobile1/test/gettyimage/holder/GettyViewProgressBarHolder.java
package com.rsupport.mobile1.test.gettyimage.holder;

import android.view.View;
import android.widget.ProgressBar;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.rsupport.mobile1.test.R;

public class GettyViewProgressBarHolder extends RecyclerView.ViewHolder {
    public final View parent;
    public final TextView tv;
    public final ProgressBar pb;

    public GettyViewProgressBarHolder(@NonNull View itemView, int viewType) {
        super(itemView);
        parent= itemView.findViewById(R.id.parent);
        tv= itemView.findViewById(R.id.tv);
        pb= itemView.findViewById(R.id.pb);
    }
}

File: app/src/main/java/com/rsupport/mobile1/test/gettyimage/holder/GettyViewProgressBarHolder.kt
package com.rsupport.mobile1.test.gettyimage.holder

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import android.widget.ProgressBar
import com.rsupport.mobile1.test.R

class GettyViewProgressBarHolder(itemView: View, viewType: Int) :
    RecyclerView.ViewHolder(itemView) {
    val parent: View
    val pb: ProgressBar

    init {
        parent = itemView.findViewById(R.id.parent)
        pb = itemView.findViewById(R.id.pb)
    }
}